tests/Agent/IntegrationTests/IntegrationTests/Owin/OwinMiddlewareExceptionTests.cs

```diff
@@ -83,12 +83,12 @@ public void Test()
             NrAssert.Multiple(
                 () => Assertions.MetricsExist(expectedMetrics, metrics),
                 () => Assert.NotEmpty(errorTraces),
-                () => Assert.NotEmpty(_fixture.AgentLog.GetErrorEvents()),
+                () => Assert.NotEmpty(_fixture.AgentLog.GetErrorEventPayloads()),
```

Comment:
probably could use local `errorEvents` here.

Reply:
Good catch, I missed that.